kmm/kmm-feed/src/commonMain/kotlin/com/gchristov/newsfeed/kmmfeed/FeedViewModel.kt

```diff
@@ -26,6 +26,14 @@ class FeedViewModel(
     init {
         loadNextPage()
         observeSearchQuery()
```

Comment:
Here is the key moment from the slack thread. We can launch the coroutine to load initial state and move the call to loadNextPage within the same coroutine block after the initial state is set. That should avoid the race condition I believe.

Reply:
Implemented as discussed